tests/bson/bson-encode-002.phpt

```diff
@@ -36,38 +36,38 @@ class NumericArray implements BSON\Serializable, BSON\Unserializable
 }
 
 echo "Testing top-level AssociativeArray:\n";
-$bson = BSON\fromArray(new AssociativeArray);
-echo BSON\toJSON($bson), "\n";
+$bson = fromPHP(new AssociativeArray);
+echo toJSON($bson), "\n";
 echo "Encoded BSON:\n";
 hex_dump($bson);
-$value = BSON\toArray($bson, array("document" => 'AssociativeArray'));
+$value = toPHP($bson, array("root" => 'AssociativeArray'));
```

Comment:
The test cases, of which there are four, are the same:

 * Testing top-level AssociativeArray
 * Testing embedded AssociativeArray
 * Testing top-level NumericArray
 * Testing embedded NumericArray

For the first case here, "document" would have previously not applied to anything and the decoded BSON would have been an array (since changed to stdClass in this PR). Given the name of the test, I feel it should actually be testing "root", which didn't exist at the time it was originally created.

Otherwise, I would have simply changed the expected output from array to stdClass. That assertion is effectively covered by the very next "Testing embedded AssociativeArray" case.